Runtime/ApiRequests.cs

```diff
@@ -68,11 +68,11 @@ public static async Task<GetImagineResult> GetRequestStatusAsync(string imagineO
             return await GetAsync<GetImagineResult>("imagine/requests/obfuscated-id/" + imagineObfuscatedId, apiKey);
         }
 
-        public static async Task<Texture2D> DownloadTextureAsync(string textureUrl)
+        public static async Task<Texture2D> DownloadTextureAsync(string textureUrl, bool readable = false)
```

Comment:
imho this should probably always be true in our case

Reply:
It's not generally needed at runtime when we can use a compute shader